src/main/java/com/karankumar/bookproject/backend/util/DateUtils.java

```diff
@@ -0,0 +1,39 @@
+package com.karankumar.bookproject.backend.util;
+
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.time.temporal.WeekFields;
+import java.util.Locale;
+
+@Component
+public class DateUtils {
+
+    private static final int WEEKS_IN_YEAR = 52;
+
+    /**
+     * @return the current week number of the year
+     */
+    public int getWeekOfYear() {
+        LocalDate now = LocalDate.now();
```

Comment:
This method can be static as well, advantage of that is we can avoid creating an instance of the class?

Reply:
Agreed